contracts/dnsregistrar/DNSRegistrar.sol

```diff
@@ -104,26 +97,26 @@ contract DNSRegistrar is IDNSRegistrar {
         }
     }
 
-    function supportsInterface(bytes4 interfaceID) external pure returns (bool) {
-        return interfaceID == INTERFACE_META_ID ||
+    function supportsInterface(bytes4 interfaceID) external pure override returns (bool) {
+        return interfaceID == type(IERC165).interfaceId ||
                interfaceID == type(IDNSRegistrar).interfaceId;
     }
 
-    function _claim(bytes memory name, bytes memory proof) internal returns(bytes32 rootNode, bytes32 labelHash, address addr) {
+    function _claim(bytes memory name, bytes memory data) internal returns(bytes32 rootNode, bytes32 labelHash, address addr) {
```

Comment:
What's the reason for changing the name of this variable name from proof to data?

Reply:
For consistency with `proveAndClaim`.